vault/core.go

```diff
@@ -2157,11 +2160,6 @@ func (c *Core) postUnseal(ctx context.Context, ctxCancelFunc context.CancelFunc,
 			c.logger.Warn("post-unseal post seal migration failed", "error", err)
 		}
 	}
-	err := c.HandleVersionTimeStamps(c.activeContext)
-	if err != nil {
-		c.logger.Warn("post-unseal version timestamp setup failed", "error", err)
```

Comment:
Is this the extra log line you were talking about in the PR description, where essentially the standby would always warn about the timestamp loading/storing not succeeding in the logs?

Reply:
I didn't actually test and see the log line myself, I speculated that we'd see this line on perf standbys.